src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ControllerResponse = {
+  isSuccess: number;
+  statusCode: number;
+  message: string;
+  data?: any;
+  error?: any;
```

Comment:
PullProofAI

Consider using specific types instead of `any` for `data` and `error` in the `ControllerResponse` interface to improve type safety and maintainability.
```diff
- data?: any;
+ data?: SpecificType;
- error?: any;
+ error?: SpecificErrorType;
```


<!-- This is an auto-generated comment by PullProofAI -->